### PR TITLE
If we fail to parse a response, log the uri

### PR DESCRIPTION
**Pull request by Hellowlol**
and the original response

### CODE REVIEW

**Comment by Hellowlol** [comment]
We try again :D

File: plexpy/http_handler.py [hunk 1]
```diff
@@ -88,19 +88,25 @@ def make_request(self,
...
+                    else:
+                        output = request_content
+    
+                    if return_type:
+                        return output, content_type
+    
+                    return output
+                    
+                except Exception as e:
+                    logger.warn(u"Failed format response from uri %s to %s error %s. The original response was %s" % (uri, output_format, e, request_content))
```
Comment (drzoidberg33) [inline plexpy/http_handler.py]: Not so sure we should log `request_content` here. That may be a very large amount of text.

Comment (Hellowlol) [inline plexpy/http_handler.py]: Yes, but the its easy to see where is went wrong then. But its up to you. We should change the memory log too, its like having a memory leak.

But your the man, if you want it removed, let me know.

Comment (drzoidberg33) [inline plexpy/http_handler.py]: Ye, I don't want an entire xml dump being pushed into one line of the log. If you want to dump the xml then rather dump it to a separate file.

**Comment by Hellowlol** [comment]
done

File: plexpy/http_handler.py [hunk 2]
```diff
@@ -88,19 +90,25 @@ def make_request(self,
...
+
+                    if return_type:
+                        return output, content_type
+
+                    return output
 
-                if return_type:
-                    return output, content_type
+                except Exception as e:
+                    logger.warn(u"Failed format response from uri %s to %s error %s" % (uri, output_format))
```
Comment (drzoidberg33) [inline plexpy/http_handler.py]: You're missing an argument here. `e` is not being passed.

Comment (Hellowlol) [inline plexpy/http_handler.py]: lol, sorry, it got late yesterday.

**Pull request merged**